pkg/gardenlet/operation/botanist/vpnshoot.go

```diff
@@ -33,7 +34,7 @@ func (b *Botanist) DefaultVPNShoot() (component.DeployWaiter, error) {
 		HighAvailabilityEnabled:              b.Shoot.VPNHighAvailabilityEnabled,
 		HighAvailabilityNumberOfSeedServers:  b.Shoot.VPNHighAvailabilityNumberOfSeedServers,
 		HighAvailabilityNumberOfShootClients: b.Shoot.VPNHighAvailabilityNumberOfShootClients,
-		SeedPodNetwork:                       b.Seed.GetInfo().Spec.Networks.Pods,
+		SeedPodNetworkV4:                     b.Seed.GetInfo().Spec.Networks.Pods,
```

Comment:
How does this work for an IPv6-only single-stack seed?

Reply:
see above. Changing back to `SeedPodNetwork` for less confusion. Filtering for IPv4 vs IPv6 is done on VPN side. Gardener will just forward the values as-is.